p18.py

```diff
@@ -0,0 +1,10 @@
+#https://leetcode.com/problems/excel-sheet-column-number/
+class Solution:
+    def titleToNumber(self, s: str) -> int:
+        d={'A': 1, 'B': 2, 'C': 3, 'D': 4, 'E': 5, 'F': 6, 'G': 7, 'H': 8, 'I': 9, 'J': 10, 'K': 11, 'L': 12, 'M': 13, 'N': 14, 'O': 15, 'P': 16, 'Q': 17, 'R': 18, 'S': 19, 'T': 20, 'U': 21, 'V': 22, 'W': 23, 'X': 24, 'Y': 25, 'Z': 26}
+        if len(s)==1:
+            return d[s[0]]
+        r=d[s[0]]
+        for i in range(len(s)-1):
+            r=r*26+d[s[i+1]]
+        return r
```

Comment:
You can calculate the corresponding digit by `ord(s[i])-64` . This would eliminate the dictionary